portfolio/src/test/java/com/google/sps/servlets/TestingTest.java

```diff
@@ -37,11 +38,64 @@ public void testMockito() {
   }
 
   
-  /**
+  
   @Test
   public void testEmail() {
-    JournalServlet email = new JournalServlet();
-    Assert.assertEquals("chiamaka@google.com", email.setEmail());
+
+    Assert.assertEquals("chiamaka@google.com", JournalServlet.setEmail());
+  }
+
+  @Test
+  public void everyEntityHasSalienceScore() {
+    
+    List<String> test = new ArrayList();
+    test.add("string");
+    test.add("0.0");
+    test.add("string");
+    test.add("string");
+
+    List<String> expected = new ArrayList();
+    expected.add("Unequal amount of subject, type, and salience");
+
+    Assert.assertEquals(expected, JournalServlet.getSubjects(test));
+  }
+
+  @Test
+  public void catchUnequalAttributes() {
+    
+    List<String> test = new ArrayList();
+    test.add("string");
+    test.add("0.0");
+    test.add("string");
+
+    List<String> expected = new ArrayList();
+    expected.add("Unequal amount of subject, type, and salience");
+
+    Assert.assertEquals(expected, JournalServlet.getSubjects(test));
+  }
+  
+  /**
+  @Test
```

Comment:
Delete all the commented code here and in other files.  With VCS there's no need to keep around old code